community/kernel/src/main/java/org/neo4j/kernel/impl/store/format/StoreVersions.java

```diff
@@ -36,27 +38,43 @@
     // Enterprise formats
     public static final String HIGH_LIMIT_V3_0 = "vE.H.0";
 
+    private static final List<String> communityStoreVersions = Arrays.asList(
```

Comment:
Now those 2 list will need to be always updated when new format will be introduced and potentially can cause fragile behaviour.
Also it has nothing to do with this PR